simphony/cuds/material.py

```diff
@@ -0,0 +1,31 @@
+import uuid
+
+from simphony.core.data_container import DataContainer
+
+
+class Material(object):
+    """A class that represents a material.
+
+    Class describes a material and its data.
+
+    Attributes
+    ----------
+    description : str
+        textual description of the material
+    uid : uuid.UUID
+        the id of the material. If None, then it is generated
+    data : DataContainer
+        data associated with this material
+
+    """
+    def __init__(self, uid=None, data=None, description=""):
```

Comment:
> I think that SSB expects that user can save a name parameter to the material with
material_foo = Material(name="foo")
(see #227)
Maybe there should be a name argument together with (optional) description

#227 had a mistake by adding the name.   that said, we could use 'name' instead of 'uid' for our unique definition.  it does seem easier for the user to use.

I am personally indifferent to either using a unique string 'name' or a 'uid' (but not both)  

(we wrote on the wiki that for MaterialRelation we would use a 'name' ..see faq at the end of https://publicwiki-01.fraunhofer.de/SimPhoNy-Project/index.php/MaterialRelations_proposal)